test-harness/index.ts

```diff
@@ -45,8 +45,7 @@ describe('harness', () => {
 
     test(testText, done => {
       const [command, ...args] = parsed.command.split(' ').map(t => t.trim());
-      const serverArgs = [...parsed.server.split(' ').map(t => t.trim()), tmpFileHandle.name];
-
+      const serverArgs = parsed.server.split(' ').map(t => t.trim().replace('${file}', tmpFileHandle.name));
```

Comment:
```suggestion
      const serverArgs = parsed.server.split(/ +/).map(t => t.replace('${file}', tmpFileHandle.name));
```
^ this could be "more-what-author-wanted"

Reply:
What does that regex do?